src/azul/plugins/metadata/hca/aggregate.py

```diff
@@ -47,20 +51,31 @@
 class HCAAggregate(Aggregate):
 
     @cached_property
-    def total_estimated_cells(self) -> int:
+    def cell_count(self) -> int:
         cs: JSON
         return sum(cs['total_estimated_cells']
                    for cs in self.contents['cell_suspensions']
                    if cs['total_estimated_cells'] is not None)
 
+    @cached_property
+    def effective_cell_count(self) -> int:
+        if self.entity.entity_type == 'projects':
+            project = one(self.contents['projects'])
+            project_cells = project['estimated_cell_count'] or 0
+            return max(project_cells, self.cell_count)
+        else:
+            return self.cell_count
+
     @classmethod
     def field_types(cls, field_types: FieldTypes) -> FieldTypes:
         return dict(super().field_types(field_types),
-                    total_estimated_cells=pass_thru_int)
+                    cell_count=pass_thru_int,
```

Comment:
I don't think this is the right field type. Your test demonstrates that both of these can be None and we don't use pass through types unless we are sure they that either can't be None or we don't need to filter by them.